commands/debug/version.js

```diff
@@ -0,0 +1,33 @@
+const Commando = require("discord.js-commando");
+
+
+class Version extends Commando.Command {
+
+    constructor(client) { 
+        super(client, {
+            name: "version",
+            group: "debug",
+            memberName: "version",
+            description: "version",
+            guildOnly: true
+        });
+    }
+
+    /**
+     * Method that executes on invocation of command.
+     * @param {String} message 
+     * @param {String} args 
+     */
+    async run(message, args) {
+        let version = process.env.npm_package_version;
+        
+        if (!version) {
+            console.warn("Version not available from process. Using package.json instead. Try running bot with \"npm start\"");
+            version = require('../../package.json').version;
```

Comment:
Possible security implications with `require('../../package.json')`. See here https://stackoverflow.com/questions/9153571/is-there-a-way-to-get-version-from-package-json-in-nodejs-code#comment55397717_10855054

Reply:
Might not be an issue since Browserify is currently not being used.

Reply:
Not an issue without Browserify, however let's add a comment here with this security concern and a link to this answer.